### DISCUSSION TITLE
Closing button for Modal Leaderboard

### DISCUSSION THREAD

**Discussion by ShuhaibKVL** [monkeytypegame/monkeytype#5958] [Ideas] [answered]
When clicking on the Leaderboard navigation, the modal opens and takes up almost the entire screen, making it hard for users to understand that it’s a modal. It takes them a moment to figure out how to go back. Adding a clear close button or better highlighting the fact that it’s a modal would improve user experience and prevent users from getting stuck.

![Screenshot 2024-10-14 125628](https://github.com/user-attachments/assets/96993594-c213-4487-88b0-6a9938401815)

**Comment by fehmer** [comment, marked as answer]
The leaderboard will be re-designed soon and likely not longer use a modal.
